Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2603,6 +2603,52 @@ namespace OpenInfraPlatform
 
 					return carve::geom::VECTOR(x, y, 0.);
 				}
+#endif
+
+
+				/**********************************************************************************************/
+				/*! \brief Calculates a direction of the curve.
+				* \tparam TCurve					A type of the curve.
+				* \param[in] curve					A pointer to data from the curve.
+				* \param[in] trimmingVec			A vector of pointers to data form \c IfcTrimmingSelect.
+				* \param[in] trimmingPreference		Specifies the preferred way of trimming.
+				* \return							The direction of the curve. 
+				*/
+				template <typename TCurve>
+				carve::geom::vector<3> getDirectionOfCurve(
+					const EXPRESSReference<TCurve>& curve,
+					const std::vector<std::shared_ptr<typename IfcEntityTypesT::IfcTrimmingSelect>>& trimmingVec,
+					const typename IfcEntityTypesT::IfcTrimmingPreference& trimmingPreference
+				) const noexcept(false)
+				{
+					std::vector<carve::geom::vector<3>> points;

```

Comment:
since we are calculating directions, it would be better to also call them directions :) (not points)

Reply:
Resolved: 4192532